packages/ui-img/src/Img/theme.js

```diff
@@ -22,9 +22,21 @@
  * SOFTWARE.
  */
 
-export default function generator() {
+/**
+ * Generates the theme object for the component from the theme and provided additional information
+ * @param  {Object} theme The actual theme object.
+ * @param  {Object} themeOverride User provided overrides of the default theme mapping.
+ * @return {Object} The final theme object with the overrides and component variables
+ */
+const generateComponentTheme = (theme, themeOverride = {}) => {
+  const componentVariables = {
+    effectTransitionDuration: '1s'
+  }
+
   return {
-    effectTransitionDuration: '1s',
-    imageBlurAmount: '0.25em'
```

Comment:
imageBlurAmount missing

Reply:
I thought I can get way with it. I didn't think it through properly